packages/gql-executor/gqlExecutor.ts

```diff
@@ -8,7 +8,6 @@ import '../server/initSentry'
 import RedisStream from './RedisStream'
 
 tracer.init({
-  enabled: process.env.DD_TRACE_ENABLED === 'true',
```

Comment:
i kept getting these typescript errors (from a DD upgrade awhile ago I believe...). DD grabs the env var directly, we can no longer pass it in